### ISSUE TITLE
Request to remove the remaining datacap

### ISSUE THREAD

**Issue by MikeH1999** [filecoin-project/Allocator-Registry#13]
Failure to follow the process own have set up，reference ：https://github.com/MikeH1999/RFfil/issues/3

Delete the remaining datacap with the following wallet
f1tnkhku73afy6emaggffwewacjyqjnyn5feubeay

**Comment by galen-mcandrew** [comment]
@willscott & @kevzak this is a good example of a new DataCap remove request. Can we test the netti tooling? Do we have additional templates or details for DataCap removal requested by an allocator?

**Comment by willscott** [comment]
@galen-mcandrew  as soon as you complete https://github.com/filecoin-project/notary-governance/issues/1105 you'll be able to use that tooling to complete this

**Comment by kevzak** [comment]
> @willscott & @kevzak this is a good example of a new DataCap remove request. Can we test the netti tooling? Do we have additional templates or details for DataCap removal requested by an allocator?

yes, this is one (Mike) of the four allocators who assigned 1PiB in previous weeks. All 4 have stated (direct message to me) they would like the DataCap removed. Here is a slack with their information:

https://filecoinproject.slack.com/archives/C06MTBZ44P2/p1711652838671219

**Comment by nicelove666** [comment]
Yes, I have contacted @kevzak , at the time, we had requested the client to stop using LDN and close the application.

As for why there was a 1PiB allocation in the first round, in v3.1, when the client applied for 1PiB as a weekly allocation, they could only receive 512TiB. I think that in V5, even if the client applies for 1PiB, they will still only receive 512T.Furthermore, at that time, the signature website could not modify the data amount, and we deeply apologize for the mistake that caused.

Now, https://allocator.tech/ has been updated, and regardless of how much the client applies for weekly, we can manually adjust the quantity ourselves. Thanks @willscott  for your hard work.

Currently, we have set the first round to 100TiB to provide feedback on this situation.

Finally, a salute to everyone's hard work @galen-mcandrew  @Kevin-FF-USA . Let's meet in HK web3.
